tests/serve/admin/test_storage.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+from openapi_typed_2 import convert_to_OpenAPIObject
+from tornado.httpclient import HTTPClientError, HTTPRequest
+
+from meeshkan.serve.admin import make_admin_app
+from meeshkan.serve.mock.scope import Scope
+from meeshkan.serve.mock.specs import OpenAPISpecification
+from tests.util import spec_dict
+
+scope = Scope()
+
+
+@pytest.fixture
+def app(mock_data_store, rest_middleware_manager):
+    return make_admin_app(scope, mock_data_store, rest_middleware_manager)
+
+
+@pytest.fixture(autouse=True)
+def setup():
+    yield True
+    scope.clear()
+
+
+@pytest.mark.gen_test
+def test_storage(mock_data_store, http_client, base_url):
```

Comment:
What does this test in the storage? Maybe split it into smaller tests that document how it's used?

Reply:
It's easier that way. Because before resetting storage we have to modify it. So we test clearing and resetting together.